src/lib/parsers/Args.ts

```diff
@@ -45,16 +45,16 @@ export class Args {
 	/**
 	 * The internal Lexure parser.
 	 */
-	protected readonly parser: Lexure.ArgumentStream;
+	protected readonly parser: ArgumentStream;
```

Comment:
Parser should have a default value

Reply:
https://github.com/sapphiredev/utilities/pull/404/files#diff-63b98d6beb14a27a657df5479d7a87f5136e762688cc29c9f4543a96ac8a492aR64